panel/src/components/Forms/Writer/Extensions.js

```diff
@@ -92,7 +92,7 @@ export default class Extensions {
 					buttons[button.id ?? button.name] = button;
 				}
 			} else {
-				buttons[extension.name] = extension.button;
+				buttons[extension.name] = { name: extension.name, ...extension.button };
```

Comment:
`Explanation:` helps for plugins that might not repeat the name in the button definition, but have a `get name()` in place. Without the name in the button definition, `isNodeActive` won't be able to match them.